### PR TITLE
Fixes #96 Package version on AppVeyor not incremented

### LINKED ISSUE #96: Package version on AppVeyor not incremented

### PR DESCRIPTION
**Pull request by Yuri05**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ install:
   - git submodule update --init --recursive
 
 environment:
-  app_version: "1.2.0"
+  app_version: "1.2"
```
Comment (Yuri05) [inline appveyor.yml:16]: 3rd version number is appended as %APPVEYOUR_BUILD_VERSION%, e.g. 1.2.8 for this PR

File: rakefile.rb [hunk 2]
```diff
@@ -1,197 +1,29 @@
-require 'open-uri'
```
Comment (Yuri05) [inline rakefile.rb:1]: seems the rakefile.rb was copied from ospsuite-R repo. Things like installing PK-Sim, creating special linux builds etc. are not required here, only the version replacement in the DESCRIPTION file before build.
So in fact this rakefile.rb is copied from the tlf repo :)

Comment (IndrajeetPatil) [reply rakefile.rb:1]: Thanks for the cleanup! Don't know Ruby, so didn't want to break something unintentionally :)

File: appveyor.yml (hunk 1 above)
Comment (IndrajeetPatil) [reply appveyor.yml:16]: Cool.

**Review by IndrajeetPatil — APPROVED** [review]

**Pull request merged**